packages/ui/src/ui/pages/query-tracker/module/query/actions.ts

```diff
@@ -128,10 +130,16 @@ export function loadQuery(
                 dispatch(loadCliqueByCluster(query.settings.cluster as string));
             }
 
+            const queryItem = prepareQueryPlanIds(query);
+
+            if (saveQuery) {
```

Comment:
The name of the parameter is confusing. It looks like `loadQuery` ignores response when `saveQuery === true`, doesn't it?